src/main/java/net/imagej/ops/topology/eulerCharacteristic/EulerCharacteristic26N.java

```diff
@@ -31,12 +31,13 @@
  * Bone 14: 173-182.
  * <a href="http://dx.doi.org/10.1016/8756-3282(93)90245-6">doi:10.1016/8756-3282(93)90245-6</a>
  *
- * @author Richard Domander
- * @author David Legland (original MatLab implementation)
+ * @author Richard Domander (Royal Veterinary College, London)
+ * @author David Legland  - original MatLab implementation
```

Comment:
It somehow makes me really happy that you changed this to a dash, so that you wouldn't have two inconsistent meanings for parenthetical author notes (one meaning "affiliated with this place" and another meaning more generally "additional details of authorship"). 👍